pluralize.js

```diff
@@ -222,6 +222,27 @@
     pluralRules.push([sanitizeRule(rule), replacement]);
   };
 
+  /**
+   * Remove a pluralization rule from the collection.
+   *
+   * @param {(string|RegExp)} rule
+   */
+  pluralize.removePluralRule = function (rule) {
```

Comment:
Do you think we should match both the rule and the result, or just the rule itself? This seems reasonable, just want to see if you've thought about this case.

Reply:
I'd say just the rule. It's what's 'matched' when searching for a replacement, not both.